### PR TITLE
Marine planner

### PR DESCRIPTION
**Pull request by rcranley**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,22 @@
 Vagrantfile.local
 /docs/_build/
 /db.sqlite3
+madrona-analysistools
```
Comment (edrex) [inline .gitignore]: seems like these per-component lines aren't needed since they're all in src or deps

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,22 @@
...
+madrona-scenarios
+madrona-user-profile
+mp-clipping
+mp-data-manager
+mp-drawing
+mp-explore
+mp-profile
+mp-visualize
+p97-nursery
+src
```
Comment (edrex) [inline .gitignore]: change to `deps`

File: README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ machine, exported to the VM as a shared folder; code editing and Git operations
 
  - Clone the repo
  - Copy `marco_config/settings/local.template.py` as `local.py` and customize.
- - `pip install -r requirements.txt`
+ - `pip install --src ./src -r requirements.txt`
```
Comment (edrex) [inline README.md]: change to `./deps`

File: marco_site/static/scss/_marine_planner.scss [hunk 4]
```diff
@@ -0,0 +1,478 @@
+.navbar{
```
Comment (edrex) [inline marco_site/static/scss/_marine_planner.scss]: wrap this whole file in a view-specific body selector like

`body.template-visualize{}`

Will need to add view specific body class like:

`{% block body_class %}template-visualize{% endblock %}`

(see `base.html`)

File: marco_site/static/scss/_marine_planner.scss [hunk 5]
```diff
@@ -0,0 +1,478 @@
...
+#print-modal .modal-body {
+    overflow: hidden;
+}
+#print-modal img.thumb {
+    display: block;
+    margin: auto;
+    width: 100%;
+}
+
+body.print .tabs,
```
Comment (edrex) [inline marco_site/static/scss/_marine_planner.scss]: with above wrapper selector, will need to replace all these `^body.`s with `&.` (see [sass docs](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#parent-selector))

**Comment by rcranley** [comment]
@edrex Updated per your comments / our discussions from earlier. 
Have a look.

**Comment by edrex** [comment]
+1 to merge

**Pull request merged**